sdk/tools/xml2sdb/xml2sdb.cpp

```diff
@@ -288,6 +288,17 @@ bool WriteGeneric(PDB pdb, std::list<T>& data, Database& db)
     return true;
 }
 
+template<typename T>
+bool WriteGeneric(PDB pdb, std::list<T>& data, Database& db, TAG tag_type)
+{
+    for (typename std::list<T>::iterator it = data.begin(); it != data.end(); ++it)
```

Comment:
I think all our compilers let you do `auto it` now?

Reply:
Why breaking existing code on purpose?

Reply:
From what I can see this is neither "existing code" nor "breaking" anything.

Reply:
It is new code, and replace 'breaking existing code' with 'make code readable'.

Reply:
maybe we can even use range based for loops :)